tests/test_common.c

```diff
@@ -8,6 +8,8 @@
  * Module contains functions shared between functional/ and unit/ test sources.
  * -----------------------------------------------------------------------------
  */
+#include <unistd.h>
```

Comment:
Why?

Reply:
don't recall why it was added. Took it out ... builds seem to work. Will re-test in CI.